hmt_escrow/eth_bridge.py

```diff
@@ -232,6 +238,100 @@ def deploy_factory(gas: int = GAS_LIMIT, **credentials) -> str:
     return contract_addr
 
 
+def get_pub_key_from_addr(wallet_addr: str) -> bytes:
+    """
+    Given a wallet address, uses the kvstore to pull down the public key for a user
+    in the hmt universe, defined by the kvstore key `hmt_pub_key`.  Works with the
+    `set_pub_key_at_address` function.
+
+    Requires that the `GAS_PAYER` environment variable be set to the
+    address that will be paying for the transaction on the ethereum network
+
+    Args:
+        wallet_addr (string): address to get the public key of
+
+    Returns:
+        bytes: the public key in bytes form
+
+    >>> import os
+    >>> from web3 import Web3
+    >>> get_pub_key_from_addr('badaddress')
+    Traceback (most recent call last):
+      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
+        compileflags, 1), test.globs)
+      File "<doctest __main__.get_pub_key_from_addr[2]>", line 1, in <module>
+        get_pub_key_from_addr('blah')
+      File "hmt_escrow/eth_bridge.py", line 268, in get_pub_key_from_addr
+        raise ValueError('environment variable GAS_PAYER required')
+    ValueError: environment variable GAS_PAYER required
+    >>> os.environ['GAS_PAYER'] = "0x1413862C2B7054CDbfdc181B83962CB0FC11fD92"
+    >>> os.environ['GAS_PAYER_PRIV'] = "28e516f1e2f99e96a48a23cea1f94ee5f073403a1c68e818263f0eb898f1c8e5"
+    >>> set_pub_key_at_addr('blah')  #doctest: +ELLIPSIS
```

Comment:
Let's go with `b"2dbc2c2c86052702e7c219339514b2e8bd4687ba1236c478ad41b43330b08488c12c8c1797aa181f3a4596a1bd8a0c18344ea44d6655f61fa73e56e743f79e0d"` instead of `'blah'` as this is documentation at the same time.

Reply:
done